streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/service/StreamCatalogService.java

```diff
@@ -1108,7 +1108,7 @@ public CustomProcessorInfo removeCustomProcessorInfoAsBundle(String name) throws
         }
         CustomProcessorInfo customProcessorInfo = new CustomProcessorInfo();
         this.removeTopologyComponentBundle(customProcessorBundle.getId());
-        return customProcessorInfo;
+        return customProcessorInfo.fromTopologyComponentBundle(customProcessorBundle);
```

Comment:
nit: Can we have this as below as created instance is not used else where?
```
new CustomProcessorInfo().fromTopologyComponentBundle(customProcessorBundle);
```

Reply:
Will do